project/server/app.js

```diff
@@ -1,26 +1,109 @@
 const express = require('express');
 const bodyParser = require('body-parser');
+const cors = require('cors');
+const IsValid = require('../src/components/validationComponent');
+const mongoose = require('mongoose');
+const MongoClient = require('mongodb');
+const Schema = mongoose.Schema;
+const users = require('./routes/users');
+const router = express.Router();
+const Users = require('./models/Users.js');
+const multer = require('multer');
+const assert = require('assert');
 
-const db = require('./utils/dataBaseUtils.js');
+const db = mongoose.connection;
+mongoose.connect('mongodb://localhost:27017/myDataBase', ((err)=>{
+    if (err) throw err;
+    console.log('Successfully connected');
+}))
 
-db.setUpConnection();
+const Validation = new IsValid();
 
 const app = express();
+app.use(cors());
+
+app.use('/users', users);
 
 app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
 
-app.get('/notes', (req, res)=>{
-  db.listNotes().then(data => res.send(data));
+const storage = multer.diskStorage({
+    destination: (req, file, cb) => {
+        cb(null, 'server/usersImg')
+    },
+    filename: (req, file, cb) => {
+        cb(null, Date.now()+'-'+file.originalname)
+    }
 });
+const upload = multer({storage: storage});
 
-app.post('/notes', (req, res)=>{
-  db.createNote(req, body).then(data => res.send(data));
-});
+app.post('/addUser', upload.single('photo'), (req, res, next) => {
+    const user = req.body;
+    const psw = Math.random().toString(36).slice(-8);
```

Comment:
Create a function generatePassword and store this code there. Here - only invoke the function.